src/h5web/providers/hsds/utils.ts

```diff
@@ -118,8 +117,9 @@ export function convertHsdsType(hsdsType: HsdsType): HDF5Type {
     case 'H5T_STRING':
       return {
         class: HDF5TypeClass.String,
-        charSet: hsdsType.charSet.endsWith('ASCII') ? 'ASCII' : 'UTF8',
-        length: hsdsType.length,
+        charSet: hsdsType.charSet.endsWith('ASCII') ? 'ASCII' : 'UTF-8',
```

Comment:
Note that I'm storing `UTF-8` internally, instead of the `UTF8` returned by HSDS.